code/service/src/test/scala/com/namely/chiefofstate/helper/ExecutionContextHelper.scala

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2020 Namely Inc.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+package com.namely.chiefofstate.helper
+
+import scala.concurrent.ExecutionContext
+
+trait ExecutionContextHelper {
+  implicit val ec: ExecutionContext = ExecutionContext.global
```

Comment:
I wouldn't do this, just have your test set it explicitly.

Reply:
I just felt this was cleaner